x/auth/module.go

```diff
@@ -64,6 +65,10 @@ func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, rtr *mux.Rout
 	rest.RegisterRoutes(clientCtx, rtr, types.StoreKey)
 }
 
+// RegisterGRPCRoutes registers the capability module's REST service handlers.
```

Comment:
update godoc. Does this have to do with REST?

Reply:
No, will update.